src/_premake_init.lua

```diff
@@ -1336,6 +1336,19 @@
 		}
 	}
 
+	p.api.register {
+		name = "structmemberalign",
+		scope = "config",
+		kind = "string",
```

Comment:
I would just make this an integer, and set ```allowed = { 1,2,4,8,16 }```

Reply:
I agree, other than this, the PR is good to merge.